main/static/src/style/style.less

```diff
@@ -3,6 +3,9 @@
 @import "../../ext/lesshat/build/lesshat-prefixed";
 @import "../../ext/nprogress/nprogress";
 
+@import "../../ext/bootswatch/flatly/variables";
+@import "../../ext/bootswatch/flatly/bootswatch";
+
```

Comment:
All the things tagged as `example` won't be merged so I'm not planning to do anything more fancy, especially with the admin area as it will require building 10+ different CSS and in real life situations you have only one theme that is none of the these ones.. (at least you shouldn't launch a product with a default theme)

P.S. As these are the examples eventually I'll clean up the comments.. take no offence..